docs/user_guide/thresholder.rst

```diff
@@ -0,0 +1,560 @@
+How to use it
+~~~~~~~~~~~~~
+
+The group-specific thresholds are specified in :code:`threshold_dict`. This is
+a dictionary where the keys are the sensitive feature value of the sub-group,
+and the values corresond to the threshold. Below, we will show all different
+kind of ways to specify thresholds.
+
+Furthermore, we will plot some information about the obtained predictions,
+in order to help gain an insight in the way :class:`Thresholder` works.
+
+Dataset
+^^^^^^^^
+The dataset used in all examples below, unless specified otherwise, is a
+clincial dataset of hospital re-admissions over a ten-year period (1998-2008)
+for diabetic patients across 130 different hospitals in the US. Each record
+represents the hospital admission records for a patient diagnosed with
+diabetes whose stay lasted one to fourteen days. We would like to develop a
+classification model, which decides whether the patients should be suggested
+to their primary care physicians for an enrollment into the high-risk care
+management program. The positive prediction will mean recommendation into the
+care program.
+
+
+.. doctest:: thresholder
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> import pandas as pd
+    >>> import numpy as np
+    >>> from sklearn.model_selection import train_test_split
+    
+    >>> #Get dataset, drop columns with unknown sf values
+    >>> df = pd.read_csv("https://raw.githubusercontent.com/fairlearn/talks/main/2021_scipy_tutorial/data/diabetic_preprocessed.csv")
+
+    >>> df = df[df['gender'] != 'Unknown/Invalid']
+    >>> df = df[df['race'] != 'Unknown']
+
+    >>> Y, A = df.loc[:, 'readmit_30_days'], df.loc[:, 'race']
+
+    >>> #Create dummy's, drop A from X, drop columns that correlate with Y 
+    >>> X = pd.get_dummies(df.drop(columns=["race", "discharge_disposition_id",
+    ...                                     "readmitted",
+    ...                                     "readmit_binary",
+    ...                                     "readmit_30_days"]))
+
+    >>> X_train, X_test, Y_train, Y_test, A_train, A_test, df_train, df_test = train_test_split(
+    ...        X,
+    ...        Y,
+    ...        A,
+    ...        df,
+    ...        test_size=0.50,
+    ...        stratify=Y,
+    ...        random_state=445)
+
+    >>> #Resample datset such that training data has same number of positive and negative samples
+    
+    >>> def resample_dataset(X_train, Y_train, A_train):
+    >>>   negative_ids = Y_train[Y_train == 0].index
+    >>>   positive_ids = Y_train[Y_train == 1].index
+    >>>   np.random.seed(1)
+    >>>   balanced_ids = positive_ids.union(np.random.choice(a=negative_ids, size=len(positive_ids)))
+
+    >>>   X_train = X_train.loc[balanced_ids, :]
+    >>>   Y_train = Y_train.loc[balanced_ids]
+    >>>   A_train = A_train.loc[balanced_ids]
+    
+    >>>   return X_train, Y_train, A_train
+
+    >>> X_train, Y_train, A_train = resample_dataset(X_train, Y_train, A_train)
+   
+
+Classifier: 'Neutral' predictions
```

Comment:
By "neutral", do you mean a uniform threshold for every group? Neutral is somewhat ambiguous...

Reply:
Yes, that’s what I mean (the standard threshold of 0.5). I changed it to default now, to emphasize that this are predictions without using `Thresholder`. This does not really focus on the fact that there is now a uniform threshold, but I hope the text after the header makes this clear enough?